cartography/intel/gcp/__init__.py

```diff
@@ -286,41 +277,25 @@ def _sync_single_project(
     """
     # Determine the resources available on the project.
     enabled_services = _services_enabled_on_project(resources.serviceusage, project_id)
-    if service_names.compute in enabled_services:
-        compute.sync(neo4j_session, resources.compute, project_id, gcp_update_tag, common_job_parameters)
-    if service_names.storage in enabled_services:
-        storage.sync_gcp_buckets(neo4j_session, resources.storage, project_id, gcp_update_tag, common_job_parameters)
-    if service_names.gke in enabled_services:
-        gke.sync_gke_clusters(neo4j_session, resources.container, project_id, gcp_update_tag, common_job_parameters)
-    if service_names.dns in enabled_services:
-        dns.sync(neo4j_session, resources.dns, project_id, gcp_update_tag, common_job_parameters)
-    if service_names.cloudsql in enabled_services:
-        sql.sync_sql(neo4j_session, resources.cloudsql, project_id, gcp_update_tag, common_job_parameters)
-    if service_names.cloudbigtable in enabled_services:
-        bigtable.sync_bigtable(
-            neo4j_session, resources.cloudbigtable, project_id, gcp_update_tag,
-            common_job_parameters,
-        )
-    if service_names.cloudkms in enabled_services:
-        cloudkms.sync_kms(neo4j_session, resources.cloudkms, project_id, gcp_update_tag, common_job_parameters)
-    if service_names.cloudrun in enabled_services:
-        cloudrun.sync_cloudrun(neo4j_session, resources.cloudrun, project_id, gcp_update_tag, common_job_parameters)
-    if service_names.iam in enabled_services:
-        iam.sync(
-            neo4j_session, resources.iam, resources.crm_v1, resources.admin,
-            project_id, gcp_update_tag, common_job_parameters,
-        )
-    if service_names.apigateway in enabled_services:
-        apigateway.sync_apigateways(
-            neo4j_session, resources.apigateway,
-            project_id, gcp_update_tag, common_job_parameters,
-        )
-    if service_names.firestore in enabled_services:
-        firestore.sync_firestore(neo4j_session, resources.firestore, project_id, gcp_update_tag, common_job_parameters)
+    for request in requested_syncs:
+        if request in RESOURCE_FUNCTIONS:
+            if getattr(service_names, request) in enabled_services:
+                if request == 'iam':
```

Comment:
is there any way to avoid this? I know, we have to pass 2 objects and that's causing this extra condition.